### ISSUE TITLE
Verify that the imagelint action can properly detect Alpine images

### ISSUE THREAD

**Issue by seemiller** [vmware-tanzu/community-edition#4780]
## Bug Report

There is the possibility that the imagelint action is not properly detecting images that utilize an Alpine base image. See the fluxcd-source-controller 0.24.4 package as an example.

## Expected Behavior

When run, the imagelint process should properly detect if an image Alpine based.

## Environment Details

* Build version (`tanzu version`): 
* Deployment (Managed/Unmanaged cluster): 
* Infrastructure Provider (Docker/AWS/Azure/vSphere): 
* Operating System (client):

**Comment by harshanarayana** [comment]
@seemiller  @stmcginnis   I am not sure if this is a problem. Isn't the `fluxcd-source-controller-bundle` coming from https://github.com/vmware-tanzu/package-for-source-controller/blob/main/packages/fluxcd-source-controller/0.24.4/Dockerfile

This is not running with the same base image that upstream `fluxcd/source-controller` is running. It is using `alpine` there vs the `bullseye` here in the tanzu Org. 

```bash
FROM harbor-repo.vmware.com/dockerhub-proxy-cache/library/debian:bullseye

ARG TARGETPLATFORM
```

Pardon my ignorance or lack of understanding if this is wrong.

**Comment by harshanarayana** [comment]
```bash
❯ make imagelint                                                                                                                                                                                                                                                     80%  ▓▒░
User given Path /Users/harshnar/Work/Tesseract/Maglev/mks/k8s/community-edition
Total number of images to process: 1
Result 1 of 1 images
Status:   Fail
Image:    fluxcd/source-controller:v0.25.6
Message:  alpine image
-------------------------------  Fail  Summary  ---------------------------------------------
Total  Fail  : 0
-------------------------------  Pass  Summary  ---------------------------------------------
Total  Pass  : 0
-------------------------------  Pull Failed  Summary  ---------------------------------------------
Total  Pull Failed  : 0
-------------------------------  Not Identified  Summary  ---------------------------------------------
Total  Not Identified  : 0
-------------------------------  Summary  -----------------------------------------------
Total  Images                : 1
Total  Occurrences           : 0
Total  Pass                  : 0
Total  1  file(s) contain(s) this image
File Path: /Users/harshnar/Work/Tesseract/Maglev/mks/k8s/community-edition/addons/packages/fluxcd-source-controller/0.24.4/package.yaml
Image Position: 44 : 6

Total  Fail                  : 0
Total  Pull Failed           : 0
Total  Not Identified        : 0
[Note:All totals for Pass|Fail|Not Identified|Pull Failed are based on number of occurrences]
--------------------------------------------------------------------------------------------

exit status 1
make: *** [imagelint] Error 1
```

```bash
User given Path /Users/harshnar/Work/Tesseract/Maglev/mks/k8s/community-edition
Total number of images to process: 1
Result 1 of 1 images
Status:   Pass
Image:    projects.registry.vmware.com/tce/fluxcd-source-controller-bundle@sha256:4dc37bdc0a6188137905639cccd1938edb1ae882698b3d58a24615b44eb1fa3f
Message:  According to the image history, this is not an Alpine Image
Total  1  file(s) contain(s) this image
File Path: /Users/harshnar/Work/Tesseract/Maglev/mks/k8s/community-edition/addons/packages/fluxcd-source-controller/0.24.4/package.yaml
Image Position: 44 : 6

-------------------------------  Fail  Summary  ---------------------------------------------
Total  Fail  : 0
-------------------------------  Pass  Summary  ---------------------------------------------
Image:     projects.registry.vmware.com/tce/fluxcd-source-controller-bundle@sha256:4dc37bdc0a6188137905639cccd1938edb1ae882698b3d58a24615b44eb1fa3f
File Path: /Users/harshnar/Work/Tesseract/Maglev/mks/k8s/community-edition/addons/packages/fluxcd-source-controller/0.24.4/package.yaml
Image Position: 44 : 6

Total  Pass  : 1
-------------------------------  Pull Failed  Summary  ---------------------------------------------
Total  Pull Failed  : 0
-------------------------------  Not Identified  Summary  ---------------------------------------------
Total  Not Identified  : 0
-------------------------------  Summary  -----------------------------------------------
Total  Images                : 1
Total  Occurrences           : 1
Total  Pass                  : 1
Total  Fail                  : 0
Total  Pull Failed           : 0
Total  Not Identified        : 0
[Note:All totals for Pass|Fail|Not Identified|Pull Failed are based on number of occurrences]
--------------------------------------------------------------------------------------------
```

**Comment by seemiller** [comment]
@harshanarayana That Dockerfile was just added to fluxcd-source-controller because they discovered the Alpine base image issue. Definitely welcome any updates you might have for this!

**Comment by harshanarayana** [comment]
@seemiller 

`projects.registry.vmware.com/tce/fluxcd-source-controller-bundle:0.24.4` and `fluxcd/source-controller:v0.24.4` one is from actual upstream and the other is used by the Repo here. 

If you look at the layers of `projects.registry.vmware.com/tce/fluxcd-source-controller-bundle:0.24.4` this is clearly not a Alpine image. 

However `fluxcd/source-controller:v0.24.4` is. But the current code can't detect that because of the entrypoint issue. It can't copy the `/etc/os-release` and falsely claims this is not an alpine image. This is fixed by #4794 

Is there any other issue you are talking about ?
